_includes/release-notes/51new-features.md

```diff
@@ -0,0 +1,229 @@
+| `deviceType.isTouch`   | Boolean | Returns `true` if the user's device is a touch device.               |
+| `deviceType.isWebView` | Boolean | Returns `true` if the user's device is a WebView device.             |
+
+For a list of deprecated `Env` APIs, see: [Deprecated API Properties - `tinymce.Env`](#deprecatedapiproperties-tinymceenv).
+
+### Added new `referrer_policy` setting
+Used for setting the level of referrer information sent when loading plugins and CSS. Referrer policies can be used to:
+
+* Improve the privacy of end-users.
+* Assist with server-side filtering of cross-origin requests for {{site.productname}} resources.
+
+For information on using the `referrer_policy` setting, see: [Integration and setup options - `referrer_policy`]({{site.baseurl}}/configure/integration-and-setup/#referrer_policy).
+
+### Added a dark content css skin
+A dark CSS definition for `content_css` has been added to compliment the dark user interface skin.
+
+For example:
+
+{% include codepen.html id="dark-mode" %}
+
+For information on using the dark version of the default skin, see: [Customizing the editor UI - Skins]({{site.baseurl}}/general-configuration-guide/customize-ui/#skins).
+
+### Added border width to Table cell dialog
+The table plugin has been updated to include a **Border width** field in the **Cell Properties** dialog. The field will accept any [valid CSS length](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#Lengths).
+
+For example:
+
+<img alt="Cell Properties Dialog with new Border Width field" src="{{site.baseurl}}/images/border-width-cell-props.png" style="width:380px;border: 1px solid #036DD5" />
+
+### Changed the default `toolbar_drawer` to `floating`
+The default for the `toolbar_drawer` setting has been changed from `false` to `floating`. For information on the `toolbar_drawer` setting, see: [User interface options - `toolbar_drawer`]({{site.baseurl}}/configure/editor-appearance/#toolbar_drawer).
+
+### Icon changes
+In {{site.productname}} 5.0, the same icon (`paragraph`) was used for the toolbar buttons `visualchars` and `visualblocks`.
+To improve the user experience:
+
+* The `paragraph` icon has been renamed `visualchars` and is used for the `visualchars` toolbar button: ![`paragraph` renamed to `visualchars`]({{site.baseurl}}/images/icons/visualchars.svg)
+* A new `visualblocks` icon is used for the `visualblocks` toolbar button: ![New visualblocks icon]({{site.baseurl}}/images/icons/visualblocks.svg)
+
+For the list of icons included in {{site.productname}}, see: [Editor icon identifiers]({{site.baseurl}}/advanced/editor-icon-identifiers/).
+
+### Fixes to the positioning of inline dialogs and menus
+Fixes for inline dialogs and menus have been included to:
+
+* Position inline dialogs correctly when the page is scrolled.
+* Reposition inline dialogs and menus when resizing {{site.productname}}.
+
+## Premium Plugin changes
+
+### PowerPaste
```

Comment:
I guess DOC-421 should be mentioned if we're including that?

Reply:
Added